data/cfp.md

```diff
@@ -0,0 +1,99 @@
+## DACHFest 2018 Call for Papers
+
+Thanks for your interest in speaking at DACHFest 2018!
+
+DACHFest is a community-driven conference organized by Google Developer Groups from Germany, Austria, and Switzerland. It's taking place at the Munich Technical University on November 10–11: around 800 people are going to get together to enjoy 3 talk tracks, a lot of community-oriented activities, and opportunities for networking and making new friends. ❤️
+
+If you have any questions, do not hesitate to contact us on [Twitter](https://twitter.com/DACHFest) or at [cfp@dachfest.com](mailto:cfp@dachfest.com).
+
+### Deadlines 📅
+
+September 15: Last day to submit your application!
+
+September 30: We'll notify you of your submission status.
```

Comment:
15 days is extremely tight. do we have committed volunteers for the review process in this time?
I suggest moving the deadline to 1.09 to give a little bit more time

Reply:
I think that's the only way we can swing it. If we move the deadline to September 1, we won't have enough time to course-correct if necessary, and if we move notifications any further, we might mess our speakers' schedule or even have some people double-book.